mathml/presentation-markup/operators/dir-mo-ref.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>&lt;mo&gt; RTL direction</title>
+    <meta charset="utf-8"/>
+  </head>
+  <body>
+
+    <!-- This checks prefix form in RTL mode. -->
+    _<math dir="rtl"><mo style="-webkit-margin-start: 0em; -webkit-margin-end: 0.055555555555555em;" lspace="0em" rspace="0em">+</mo><mi>x</mi></math>_
```

Comment:
I don't think we should use --webkit prefixes in WPT tests... Also, I think we haven't really defined https://github.com/mathml-refresh/mathml/issues/14 yet, so I'm not sure it's really interoperable test. Maybe this should use mspace instead?

Reply:
You are right, it did not even work in chromium/Mozilla using --webkit-margin! Using mspace fixes it.

Reply:
It seems we have a similar test with mo-paint-lspace-rspace.html?